MaterialLib/MPL/Properties/SaturationDependentSwelling.cpp

```diff
@@ -18,7 +18,9 @@
 namespace MaterialPropertyLib
 {
 SaturationDependentSwelling::SaturationDependentSwelling(
-    std::array<double, 3> swelling_pressures,
+    std::string name,
```

Comment:
Needs clang format.

Reply:
The array arguments are formatted differently.... Not nice, but consistent ;)